README.md

```diff
@@ -14,6 +14,8 @@ The build script of crate will attempt to find the lib path of libpq using the
 following methods:
 
 * First, if the environment variable `PQ_LIB_DIR` is set, it will use its value
+* Second it will look for an environment variable in the format of `PQ_LIB_DIR_{TARGET}`
+where `{TARGET}` gets replaced by the Target environment variable set for cross-compilation
```

Comment:
These should be switched, as the current implementation prefers `PQ_LIB_DIR_{TARGET}` over `PQ_LIB_DIR` (which is fine).